modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/Checkpointer.java

```diff
@@ -452,9 +456,21 @@ private void syncUpdatedPageStores(
                     return;
                 }
 
-                fsyncDeltaFilePageStoreOnCheckpointThread(entry.getKey(), entry.getValue());
+                GroupPartitionId partitionId = entry.getKey();
+
+                FilePageStore filePageStore = filePageStoreManager.getStore(partitionId.getGroupId(), partitionId.getPartitionId());
+
+                if (filePageStore == null || filePageStore.isMarkedToDestroy()) {
```

Comment:
Let's replace

```
if (COND) continue;
REST;
```

with

```
if (!COND) {
    REST
}
```

After this transformation, lines 459-473 will become identical to lines 498-510, so a method could be extracted to avoid code duplication.

Reply:
I would not like to change the code and also this file only has 332 lines

Reply:
How can it have 332 lines if github shows lines 460-463 in this particular comment? I agree with Roman, it looks very much like code duplication

Reply:
fix code duplication